### PR TITLE
Update ovirt-engine-disaster-recovery.md

### PR DESCRIPTION
**Pull request by xandradx**
Updated progress for 100% API usage !
-

-

-

I confirm that this pull request was submitted according to the [contribution guidelines](/CONTRIBUTING.md): (please @mention yourself to sign)

This pull request needs review by: (please @mention the reviewer if relevant)

### CODE REVIEW

File: source/develop/projects/proposals/ovirt-engine-disaster-recovery.md [hunk 1]
```diff
@@ -20,7 +20,8 @@ oVirt Engine DR  is a web application, that uses Java + Ovirt API + PostgreSQL a
 ## Current Status
 
 *   This project is in [incubation](/governance/adding-a-subproject/).
-*   Last updated: Jul 13 2016
+*   Last updated: Feb 7 2016
```
Comment (mureinik) [inline source/develop/projects/proposals/ovirt-engine-disaster-recovery.md]: you meant Feb 2017, right?

File: source/develop/projects/proposals/ovirt-engine-disaster-recovery.md [hunk 2]
```diff
@@ -42,7 +43,7 @@ oVirt Engine DR, uses oVirt API and perform tasks that would require manual inte
 
 **High-level work items:**
 
-*   Remove the need to alter Engine Database directly, would requiere modification to oVirt API.
+*   oVirt API, since 3.6.10, support force options, allow us to change storage connection for DR.
```
Comment (mureinik) [inline source/develop/projects/proposals/ovirt-engine-disaster-recovery.md]: support->supports, allow->allows

**Comment by xandradx** [comment]
Thanks for the corrections !

--
Jorge Luis Andrade Escobar 
jandrade@i-t-m.com <mailto:jandrade@i-t-m.com>
jandrade@itm.gt <mailto:jandrade@itm.gt> 
Red Hat Certified Architect (RHCA) Level V
Red Hat Certified Instructor/Examiner (RHCX)
# 100-000-967
gpg: 685C 68E8 9F55 771B ED7C 3CC7 5E65 8385 88D3 09A5
Phone: +502 2305-0800 ext. 4005

http://www.i-t-m.com <http://www.i-t-m.com/>

**Comment by mureinik** [comment]
@xandradx can you push an updated patch? I'd be happy to merge it.

**Pull request merged**